gnmi_server/gnoi.go

```diff
@@ -2,168 +2,20 @@ package gnmi
 
 import (
 	"context"
-	"errors"
-	"os"
-	gnoi_system_pb "github.com/openconfig/gnoi/system"
+
 	log "github.com/golang/glog"
 	"time"
 	spb "github.com/sonic-net/sonic-gnmi/proto/gnoi"
 	transutil "github.com/sonic-net/sonic-gnmi/transl_utils"
-	io "io/ioutil"
-	ssc "github.com/sonic-net/sonic-gnmi/sonic_service_client"
 	spb_jwt "github.com/sonic-net/sonic-gnmi/proto/gnoi/jwt"
-	"github.com/sonic-net/sonic-gnmi/common_utils"
 	"google.golang.org/grpc/status"
 	"google.golang.org/grpc/codes"
 	"os/user"
 	"encoding/json"
 	jwt "github.com/dgrijalva/jwt-go"
 )
 
-func KillOrRestartProcess(restart bool, serviceName string) error {
-	sc, err := ssc.NewDbusClient()
-	if err != nil {
-		return err
-	}
-	if restart {
-		log.V(2).Infof("Restarting service %s...", serviceName)
-		err = sc.RestartService(serviceName)
-		if err != nil {
-			log.V(2).Infof("Failed to restart service %s: %v", serviceName, err)
-		}
-	} else {
-		log.V(2).Infof("Stopping service %s...", serviceName)
-		err = sc.StopService(serviceName)
-		if err != nil {
-			log.V(2).Infof("Failed to stop service %s: %v", serviceName, err)
-		}
-	}
-	return err
-}
-
-func (srv *Server) KillProcess(ctx context.Context, req *gnoi_system_pb.KillProcessRequest) (*gnoi_system_pb.KillProcessResponse, error) {
```

Comment:
>KillProcess [](http://example.com/codeflow?start=19&length=11)

Is this gNOI API removed in this PR? @isabelmsft to review.

Reply:
Should KillProcess API also be moved to gnoi_system.go? KillProcess API implementation was removed here, but unit tests for KillProcess remain

Reply:
Sure, KillProcess can be removed to gnoi_system.go as well